infrastructure/environments/prod/variables.tf

```diff
@@ -8,15 +8,7 @@ variable "RESOURCE_GROUP_PRODUCTS" {
   default     = "mpazr-rg-1023"
 }
 
-variable "CLIENT_ID" {
-  description = "Service Principal Client ID"
-}
-
-variable "CLIENT_SECRET" {
-  description = "Service Principal Client Secret"
-}
-
 variable "ENVIRONMENT" {
   description = "Environment name"
-  default     = "prod"
+  default     = "non-prod"
```

Comment:
How come you've renamed this to non-prod? 🤔 

If the clusters are now named after `var.environment` won't that cause a clash if we try to spin them both up?

Reply:
Thanks, fixed that now